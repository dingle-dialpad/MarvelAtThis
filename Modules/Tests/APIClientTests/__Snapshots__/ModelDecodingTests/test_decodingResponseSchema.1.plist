<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>code</key>
	<integer>200</integer>
	<key>data</key>
	<dict>
		<key>count</key>
		<integer>20</integer>
		<key>limit</key>
		<integer>20</integer>
		<key>offset</key>
		<integer>0</integer>
		<key>results</key>
		<array/>
		<key>total</key>
		<integer>1562</integer>
	</dict>
	<key>etag</key>
	<string>5616d2bbdfec4495120e3b473ac87ace37841048</string>
	<key>status</key>
	<string>Ok</string>
</dict>
</plist>
