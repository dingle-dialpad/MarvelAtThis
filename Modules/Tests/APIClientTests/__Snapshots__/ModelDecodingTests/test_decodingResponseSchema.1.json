{
  "data" : {
    "count" : 20,
    "cursor" : {
      "limit" : 20,
      "offset" : 0
    },
    "results" : [

    ],
    "total" : 1562
  },
  "metadata" : {
    "code" : 200,
    "etag" : "5616d2bbdfec4495120e3b473ac87ace37841048",
    "status" : "Ok"
  }
}